diff --git a/node_modules/@react-native-community/netinfo/android/src/main/java/com/reactnativecommunity/netinfo/NetInfoModule.java b/node_modules/@react-native-community/netinfo/android/src/main/java/com/reactnativecommunity/netinfo/NetInfoModule.java
index 626134a..efee6e8 100644
--- a/node_modules/@react-native-community/netinfo/android/src/main/java/com/reactnativecommunity/netinfo/NetInfoModule.java
+++ b/node_modules/@react-native-community/netinfo/android/src/main/java/com/reactnativecommunity/netinfo/NetInfoModule.java
@@ -44,9 +44,9 @@ public class NetInfoModule extends ReactContextBaseJavaModule implements AmazonF
     // the upstream method was removed in react-native 0.74
     // this stub remains for backwards compatibility so that react-native < 0.74
     // (which will still call onCatalystInstanceDestroy) will continue to function
-    public void onCatalystInstanceDestroy() {
-        invalidate();
-    }
+    // public void onCatalystInstanceDestroy() {
+    //     invalidate();
+    // }
 
     // This should have an `@Override` tag, but the method does not exist until
     // react-native >= 0.74, which would cause linting errors across versions
